RequestInfoSvcs/RequestInfoManager/CompositeRequestContainerProcessor.hpp

```diff
@@ -74,7 +74,7 @@ namespace RequestInfoSvcs
     RequestContainerProcessor* child_processor) /*throw(Exception)*/
   {
     RequestContainerProcessor_var add_processor(
-      ReferenceCounting::add_ref(child_processor));
+     ReferenceCounting::add_ref(child_processor));
```

Comment:
Why is required this change ? With incorrect space indend that we use (two spaces) ...